LayoutTests/fast/forms/option-mouseevents.html

```diff
@@ -38,8 +38,8 @@
     shouldBe("event target", event.target, target);
     shouldBe("event.pageX", event.pageX, x);
     shouldBe("event.pageY", event.pageY, y);
-    shouldBe("event.offsetX", event.offsetX, offsetX);
-    shouldBe("event.offsetY", event.offsetY, offsetY);
+    shouldBe("event.offsetX", event.offsetX, offsetX + 2);
+    shouldBe("event.offsetY", event.offsetY, offsetY + 2);
```

Comment:
I think it would be better to update the call sites of `mouseeventverify2` rather than have a special `+ 2` here.